lib/WP_Auth0_LoginManager.php

```diff
@@ -448,19 +448,28 @@ public static function get_authorize_params( $connection = null, $redirect_to =
 		// Nonce is not needed here as this is not processing form data.
 		// phpcs:disable WordPress.Security.NonceVerification.NoNonceVerification
 
-		$opts = WP_Auth0_Options::Instance();
-
-		$params = [
-			'connection'    => $connection,
-			'client_id'     => $opts->get( 'client_id' ),
-			'organization'  => $opts->get( 'organization' ),
-			'scope'         => self::get_userinfo_scope( 'authorize_url' ),
-			'nonce'         => WP_Auth0_Nonce_Handler::get_instance()->get_unique(),
-			'max_age'       => absint( apply_filters( 'auth0_jwt_max_age', null ) ),
-			'response_type' => 'code',
-			'response_mode' => 'query',
-			'redirect_uri'  => $opts->get_wp_auth0_url(),
-		];
+		$opts         = WP_Auth0_Options::Instance();
+		$customParams = trim( $opts->get( 'auto_login_params' ) ?? '' );
+		$params       = [];
+
+		if ( $customParams ) {
+			parse_str( $customParams, $params );
```

Comment:
codecov is flagging this line as not covered in tests. Is this the part where the custom params are considered to be included?